commander/migrate.go

```diff
@@ -35,6 +40,40 @@ func (c *Commander) migrateCommanderData(hubble client.Hubble) error {
 	return nil
 }
 
+func (c *Commander) syncFailedTxs(hubble client.Hubble) error {
+	failedTxs, err := hubble.GetFailedTransactions()
+	if err != nil {
+		return err
+	}
+
+	if failedTxs.Len() > 0 {
+		err = c.saveFailedTxs(failedTxs)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (c *Commander) saveFailedTxs(failedTxs models.GenericTransactionArray) error {
```

Comment:
What do you think about moving this function to storage? That way we would keep `UpdateInMultipleTransactions` private and this would be more independent of db choice.